llvm-15.0.3/lldb/tools/lldb-crash-server/GDBRemoteCommunicationServerCS.cpp

```diff
@@ -82,6 +82,7 @@ GDBRemoteCommunicationServerLLCS::RegisterPacketHandlers_LLCS() {
 
 llvm::Expected<std::unique_ptr<llvm::MemoryBuffer>>
 GDBRemoteCommunicationServerLLCS::BuildTargetXml() {
+#if 0
```

Comment:
In addition to the other comment, are these if 0 blocks needed? or not?
if they are needed for future reference, provide comment. otherwise remove them all together.

Reply:
I have removed the commented out section. We don't need this code.